tools/behavior_simulator/src/state.rs

```diff
@@ -326,9 +317,9 @@ impl Default for State {
                 six: None,
                 seven: None,
             },
-            remaining_amount_of_messages: 1200,
+            remaining_number_of_messages: 1200,
             sub_state: None,
-            hulks_team_is_home_after_coin_toss: false,
```

Comment:
Nice change, please also rename all other instances of this in the repo in the files
- `game_controller_state.rs`
- `game_controller_state_message.rs`
- `game_controller_state_filter.rs`
- `game_controller_filter.rs`

Reply:
This renaming was part of #666 and I just used the naming from that. I think since my focus here is the behavior simulator that change should be a different PR. What do you think?

Reply:
Yep okay 👍🏻